app/scicrunch_requests.py

```diff
@@ -211,10 +211,10 @@ def facet_query_string(query, terms, facets, type_map):
 
     # Add search query if it exists
     qt = ""
-    if query:
+    if query != "":
```

Comment:
I believe I made those changes after PyCharm (?) complained to me. Same for the change below. I believe @hsorby was commenting about the name of some of the variables. For instance, `t` doesn't mean anything to me indeed.